app/lib/katello/util/search.rb

```diff
@@ -48,8 +48,13 @@ def self.active_record_search_classes
         initial_list + subclass_list
       end
 
+      #need to remove all instances of when scoped search conversion completes
       def self.backend_search_classes
-        [Katello::Pool]
+        pulp_backend_search_classes
+      end
+
+      def self.pulp_backend_search_classes
+        [Katello::PuppetModule]
       end
```

Comment:
I think you actually want to remove these two classes now that you PuppetModules have been converted.